tools/cldr-code/src/main/java/org/unicode/cldr/test/DisplayAndInputProcessor.java

```diff
@@ -1289,6 +1299,14 @@ private String normalizeWhitespace(String path, String value) {
             value = PLACEHOLDER_SPACE_BEFORE.matcher(value).replaceAll("\u00A0{");
         }
 
+        // Finally, replace remaining space combinations with most restrictive type CLDR-17233
+        // If we have NNBSP U+202F in combination with other spaces, keep just it
+        value = NNBSP_AMONG_OTHER_SPACES.matcher(value).replaceAll("\u202F");
+        // Else if we have NBSP U+00A0 in combination with other spaces, keep just it
+        value = NBSP_AMONG_OTHER_SPACES.matcher(value).replaceAll("\u00A0");
+        // Else if we have THIN SPACE U+2009 in combination with other spaces, keep just it
+        value = THIN_SPACE_AMONG_OTHER_SPACES.matcher(value).replaceAll("\u2009");
+
```

Comment:
Hmmm. One more suggestion: if we have all ASCII space, collapse to one. 

Or is that done elsewhere?

Reply:
That appears to be already handled elsewhere...